go-controller/pkg/cni/helper_linux.go

```diff
@@ -260,6 +278,9 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 		ipStrs[i] = ip.String()
 	}
 
+	if ifInfo.IsSmartNic {
```

Comment:
add a comment saying that that the rest of the flow was already handled on the smart nic.

Reply:
also i see that the reset of the flow of this method plug the rep/veth port to ovs
then configures pod bandwidth. where are we doing that in case of smart-nic ?

Reply:
In the ovn-node.go https://github.com/Mmduh-483/ovn-kubernetes/blob/ms-fb-reb-f/go-controller/pkg/node/node.go#L455

Reply:
`addRepPort` does not configure pod bandwidth as i see

Reply:
updated to configure the bandwidth